docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<p>Should we add one sentence explanation of different optimization level? Something like level 0 does not do any transipilation while level 3 uses a lot of transpiler passes to optimize the circuits</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Good idea, will add</p><p> </p>